src/BootstrapModalManager.tsx

```diff
@@ -36,10 +43,10 @@ export default class BootstrapModalManager extends ModalManager {
       this.adjustAndStore('paddingRight', el, size),
     );
     qsa(container, Selector.STICKY_CONTENT).forEach((el) =>
-      this.adjustAndStore('margingRight', el, -size),
```

Comment:
o wow this is just straight up a bug ooops

Reply:
Yeah, heh, was fixed separately in #5149 :)